src/NumericalAlgorithms/Interpolation/InterpolateeInitialize.hpp

```diff
@@ -0,0 +1,108 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Tags.hpp"
+#include "NumericalAlgorithms/Interpolation/InterpolateeTags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace Parallel {
+
+/// Holds Actions for Interpolator and Interpolatee.
+namespace Actions {
+
+/// Holds Actions for Interpolatee
+namespace Interpolatee {
+
+/// \ingroup ActionsGroup
+/// \brief Initializes an Interpolatee
+///
+/// Uses:
+/// - DataBox:
+///   - Tags::IndicesOfFilledInterpPoints
+///   - Tags::TemporalIds<Metavariables>
+///   - ::Tags::Domain<VolumeDim, Frame>
+///   - ::Tags::Variables<typename
+///                       InterpolateeTag::vars_to_interpolate_to_target>>
+///
+/// DataBox changes:
+/// - Adds:
+///   - Tags::IndicesOfFilledInterpPoints
+///   - Tags::TemporalIds<Metavariables>
+///   - ::Tags::Domain<VolumeDim, Frame>
+///   - ::Tags::Variables<typename
+///                       InterpolateeTag::vars_to_interpolate_to_target>>
+/// - Removes: nothing
+/// - Modifies: nothing
+///
+/// `InterpolateeTag` must contain the following type aliases:
```

Comment:
Put component into this PR, and put this documentation there.

/// For requirements on `InterpolateeTag`, see InterpolateeComponent